scripts/auth.js

```diff
@@ -0,0 +1,45 @@
+//static JSON array for users
+const users = [
+    { username: "Abd-ElSalam", password: "Abd-ElSalam" },
+    { username: "Emara", password: "Emara" },
+    { username: "Ramadan", password: "Ramadan" },
+    { username: "Hany", password: "Hany" },
+    { username: "Kiro", password: "Kiro" },
+    { username: "Hussein", password: "Hussein" }
+];
+
+//add event listner when page is loaded
+document.addEventListener("DOMContentLoaded", () => {
+    
+    //get HTML elements
+    const loginBtn = document.getElementById("loginBtn");
+    const errorText = document.getElementById("WrongPass");
```

Comment:
```suggestion
    const errorText = document.getElementById("wrongPass");
```
Please consider changing the element's ID, since it was merged as "wrongPass" in issue #7.